test/testenv/buildx.go

```diff
@@ -290,7 +290,6 @@ func (b *BuildxEnv) RunTest(ctx context.Context, t *testing.T, f TestFunc, opts
 
 	var so client.SolveOpt
 	withProjectRoot(t, &so)
-	withGHCache(t, &so)
```

Comment:
Do we have any idea why the GH cache was so slow?

Reply:
A couple of things:

1. It's just slow to transfer all that
2. The cache is going to get overwritten by each PR
3. Having separate cache scopes for each test, as is done here, means very little reuse between tests and just a bloated cache.

All in all, I found that the cache wasn't giving us any speedup.

Reply:
Thank you for the detailed answer on this, that makes a lot of sense.